src/Application/Service/MocoService.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace Ph3\DockerArch\Application\Service;
+
+use Ph3\DockerArch\Domain\Service\Model\AbstractService;
+use Symfony\Component\OptionsResolver\Options;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+/**
+ * @author Alexis NIVON <anivon@alexisnivon.fr>
+ */
+class MocoService extends AbstractService
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function getOptionsResolver(): Options
+    {
+        $resolver = new OptionsResolver();
+
+        $resolver->setRequired('mock_filename');
```

Comment:
What about `mock_file_path` name instead?

Reply:
My POV : `file_path` suggests we can pass absolute or relative path. 
But currently, data are always put into `/apps/..`.